README.md

```
@@ -43,6 +43,26 @@ Git ワークフロー自動化、GitHub レビュー管理、CI ログ分析の
 - 根拠のない指摘はドラフトに載せない
 - `review-pr` は GitHub への POST を一切行わない
 
+### レビューコメント対応の流れ
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)